src/components/Forms.js

```diff
@@ -1,12 +1,61 @@
-import './Books.css';
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { postBook } from '../redux/books/books';
 
-function Forms() {
+export default function Form() {
+  const itemId = uuidv4();
+  const [bookTitle, setTitle] = useState('');
+  const [bookAuthor, setAuthor] = useState('');
+  const [bookCategory, setCategory] = useState('');
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (bookTitle === '' || bookAuthor === '') return;
+    const newBook = {
+      item_id: itemId,
+      title: bookTitle,
+      author: bookAuthor,
+      category: bookCategory,
+    };
+    dispatch(postBook(newBook));
+    setTitle('');
+    setAuthor('');
+    setCategory('');
```

Comment:
**[OPTIONAL]** Good job with the styling but after submitting the book these input fields are not set to empty string I highly suggest you cross-check that.